react/Autosuggest/Autosuggest.test.js

```diff
@@ -50,8 +50,7 @@ describe('Autosuggest', () => {
   it('should render with mobile backdrop', () => {
     const props = getAutosuggestProps();
     props.labelProps = {
-      className: 'LABEL_TEST_CLASS',
-      someLabelProp: 'bar'
```

Comment:
This prop is pretty useless from a testing standpoint, and React v16 was actually rendering this to the underlying label when the previous version didn't. Rather than adding this pointless prop to the snapshot, I figured it'd make more sense to just remove it entirely.